buildSrc/src/main/groovy/accord.java-conventions.gradle

```diff
@@ -29,7 +29,7 @@ repositories {
 }
 
 compileJava {
-    sourceCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
we actually don't compile on jdk8... so this just makes it so we can use JDK 11 features again